include/envoy/server/options.h

```diff
@@ -65,6 +65,11 @@ class Options {
   virtual const std::string& configPath() PURE;
 
   /**
+   * @return const std::string& the path to the v2 bootstrap file.
+   */
+  virtual const std::string& bootstrapPath() PURE;
```

Comment:
Sounds good. Can you add TODO here just to let people know? Same below for other comment?